R/forecast.R

```diff
@@ -154,12 +219,61 @@ as_forecast_quantile <- function(data,
 }
 
 
-#' @rdname as_forecast
+#' @rdname as_forecast_quantile
+#' @description
+#' When creating a `forecast_quantile` object from a `forecast_sample` object,
+#' the quantiles are estimated by computing empircal quantiles from the samples
+#' via [quantile()]. Note that empirical quantiles are a biased estimator for
+#' the true quantiles in particular in the tails of the distribution and
+#' when the number of available samples is low.
+#' @param probs A numeric vector of quantile levels for which
```

Comment:
should this be called `quantile_level` or is it called probs to match `quantile`?

Reply:
Answer is yes to match quantile

Reply:
How the turntables :-D 

<img width="864" alt="image" src="https://github.com/user-attachments/assets/204c8404-6dc6-4cf9-8cc3-986d03d05104">

Yes I thought matching `quantile` made most sense here as that's what's called